### ISSUE TITLE
[Bug Report] The engine "node" is incompatible with this module. Expected version "8". Got "11.10.0"

### ISSUE THREAD

**Issue by yujinping** [ElemeFE/element#14544]
### Element UI version
2.6.0

### OS/Browsers version
MacOS Mojave/Chrome

### Vue version
2.6.7

### Reproduction Link
[https://jsfiddle.net/mmx38qxw/](https://jsfiddle.net/mmx38qxw/)

### Steps to reproduce
Upgrade from version 2.5.4
Node 11.10

```bash
MacBook-Pro:$ yarn add element-ui
yarn add v1.13.0
[1/4] 🔍  Resolving packages...
[2/4] 🚚  Fetching packages...
error element-ui@2.6.0: The engine "node" is incompatible with this module. Expected version "8". Got "11.10.0"
error Found incompatible module
info Visit https://yarnpkg.com/en/docs/cli/add for documentation about this command.
MacBook-Pro:$ node -v
v11.10.0
```

### What is Expected?
Can upgrade from 2.5.4 to 2.6 in Node 11.Support Node >=8

### What is actually happening?
error element-ui@2.6.0: The engine "node" is incompatible with this module. Expected version "8". Got "11.10.0"

**Comment by newbie78** [comment]
got the same
```
error element-ui@2.6.0: The engine "node" is incompatible with this module. Expected version "8". Got "11.5.0"
error Found incompatible module
```

**Comment by kjleitz** [comment]
Same here:

```
error element-ui@2.6.0: The engine "node" is incompatible with this module. Expected version "8". Got "10.15.2"
error Found incompatible module
```

**Comment by ziyoung** [comment]
fix #14546

**Comment by ziyoung** [comment]
https://github.com/ElemeFE/element/pull/14508#issuecomment-468712454
